### ISSUE TITLE
Add support for `mask-composite`

### ISSUE THREAD

**Issue by iamvdo** [postcss/autoprefixer#1199]
Reading this article this morning and wondering if this could be added to autoprefixer:

https://css-tricks.com/mask-compositing-the-crash-course/

Especially interesting here is the “Extending support” part

First
```css
mask-composite: add | substract | intersect | exclude
```
should add
```css
-webkit-mask-composite: source-over | source-out | source-in | xor
```
Then, cannot use old keywords in shorthand, so
```css
mask: url(...) add, url(...)
```
should add
```css
-webkit-mask: url(...), url(...);
-webkit-mask-composite: source-over;
```
Finally, if keyword is `source-in`, add `, xor` (but only if one keyword?), so
```css
mask: url(...) intersect, url(...)
```
should add
```css
-webkit-mask: url(...), url(...);
-webkit-mask-composite: source-in, xor;
```

**Comment by ai** [comment]
Seems like we can use https://caniuse.com/#feat=css-masks

**Comment by ai** [comment]
Fixed by https://github.com/postcss/autoprefixer/pull/1202

**Comment by ai** [comment]
Released in 9.5
